packages/sdk/test/search/Provider.test.tsx

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable react/display-name */
```

Comment:
Also here.

Reply:
This is for declaring the wrapper in place in line 258